core/translate/expr.rs

```diff
@@ -422,8 +376,12 @@ pub fn translate_condition_expr(
             }
         }
         ast::Expr::Parenthesized(exprs) => {
-            // TODO: this is probably not correct; multiple expressions in a parenthesized expression
-            // are reserved for special cases like `(a, b) IN ((1, 2), (3, 4))`.
+            if exprs.is_empty() {
+                // Empty parentheses should result in a parsing error.
+                crate::bail_parse_error!("Empty parenthesized expression is invalid.");
+            }
+            // I think this is always going to be len == 1...
```

Comment:
Yeah in condition expr position it should probably be a parse error to have anything other than 1 argument.